contracts/interfaces/ICover.sol

```diff
@@ -31,12 +31,12 @@ enum CoverUintParams {
 
 
 struct PoolAllocationRequest {
-  uint64 poolId;
+  uint40 poolId;
   uint coverAmountInAsset;
 }
 
 struct PoolAllocation {
-  uint64 poolId;
+  uint40 poolId;
```

Comment:
initially i had this lowered to fit the globalRewardsRatio parameter but that should be associated with the segment.

However i don't see a downside to making it smaller and potentially fitting another data field in there later.

Reply:
Have you thought about adding a minimum staking deposit when creating a pool to avoid a potential attack where a spammer depletes all possible ids?

Reply:
on the L1  gas costs should be a deterrant for that.

BUT i don't see any downside to that, it can be a fixed NXM value. 

Something like 100 NXM?